### PR TITLE
Docs: Getting support documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

## Other information

Discover Weblate, as "_Discover Weblate_" would be nice, but I think I tried that once and it wasn't in italics.
`*text*` or `_text_` should do it.

Do sites have to be running the latest version of Weblate to be listed, or just anything beyond 5.2?

### CODE REVIEW

**Comment by comradekingu** [comment]
@Fat-Zer Review time :)

File: docs/admin/support.rst [hunk 1]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 
-Listing your server without a support subsription in Discover Weblate:
+Listing your server manually:
 
-1. Register yourself at <https://weblate.org/user/>
-2. Register your Weblate server in the discovery database at <https://weblate.org/subscription/discovery/>
-3. Confirm the service activation in your Weblate and turn on the discovery listing in your Weblate management page using :guilabel:`Enable discovery` button:
+1. Register yourself at `https://weblate.org/user <https://weblate.org/user/>`_
+2. Register your Weblate site at `https://weblate.org/subscription/discovery <https://weblate.org/subscription/discovery/>`_
+3. Confirm you want to activate the service from your Weblate, and turn on "Discovery listing" from its management page by clicking :guilabel:`Enable discovery`:
```
Comment (comradekingu) [inline docs/admin/support.rst:86]:
```suggestion
3. Confirm you you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
```

File: docs/admin/support.rst [hunk 2]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
+1. Register yourself at `https://weblate.org/user <https://weblate.org/user/>`_
+2. Register your Weblate site at `https://weblate.org/subscription/discovery <https://weblate.org/subscription/discovery/>`_
+3. Confirm you you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
 
 .. image:: /images/support-discovery.png
 
 .. _customize-discover:
 
-Customizing listing
+Listing description
```
Comment (comradekingu) [inline docs/admin/support.rst:92]: This isn't great

File: docs/admin/support.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ Getting support for Weblate
 Weblate is copylefted libre software with community support.
 Subscribers receive priority support at no extra charge. Prepaid help packages are
 available for everyone. You can find more info about current support
-offerings at <https://weblate.org/support/>.
+offerings at `https://weblate.org/support <https://weblate.org/support/>`_.
```
Comment (nijel) [inline docs/admin/support.rst:8]: What is point in duplicating links? See my comments on https://github.com/WeblateOrg/weblate/pull/5742

**Comment by nijel** [comment]
> Do sites have to be running the latest version of Weblate to be listed, or just anything beyond 5.2?

Anything newer than 4.5.2 will work. This version added interface to turn on discovery on Weblate side (it has to be confirmed there to have consent of the server admin).

File: docs/admin/support.rst (hunk 3 above)
Comment (comradekingu) [reply docs/admin/support.rst:8]: They currently look like 
bilde
Not sure if the URL only makes it a hotlink, but marking it up makes it easy to change the link later on if needed.
The last / makes it easy to see what part is the actual link, and which part is the displayed name.

I'll change it to URLs only.

File: docs/admin/support.rst [hunk 4]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 .. _customize-discover:
 
-Customizing listing
+Listing description
 +++++++++++++++++++
 
-You can customize the listing by providing a text and image (570 x 260 pixels)
-at <https://weblate.org/user/>.
+Add a text and an image (570 x 260 pixels) about your Weblate site
+from `https://weblate.org/user <https://weblate.org/user/>`_.
```
Comment (comradekingu) [inline docs/admin/support.rst:96]:
```suggestion
from https://weblate.org/user.
```

File: docs/admin/support.rst [hunk 5]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 .. image:: /images/support-discovery.png
 
-Listing your server without a support subsription in Discover Weblate:
+Listing your server manually:
 
-1. Register yourself at <https://weblate.org/user/>
-2. Register your Weblate server in the discovery database at <https://weblate.org/subscription/discovery/>
-3. Confirm the service activation in your Weblate and turn on the discovery listing in your Weblate management page using :guilabel:`Enable discovery` button:
+1. Register yourself at `https://weblate.org/user <https://weblate.org/user/>`_
+2. Register your Weblate site at `https://weblate.org/subscription/discovery <https://weblate.org/subscription/discovery/>`_
```
Comment (comradekingu) [inline docs/admin/support.rst:85]:
```suggestion
2. Register your Weblate site at https://weblate.org/subscription/discovery
```

File: docs/admin/support.rst [hunk 6]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 
 .. image:: /images/support-discovery.png
 
-Listing your server without a support subsription in Discover Weblate:
+Listing your server manually:
 
-1. Register yourself at <https://weblate.org/user/>
-2. Register your Weblate server in the discovery database at <https://weblate.org/subscription/discovery/>
-3. Confirm the service activation in your Weblate and turn on the discovery listing in your Weblate management page using :guilabel:`Enable discovery` button:
+1. Register yourself at `https://weblate.org/user <https://weblate.org/user/>`_
```
Comment (comradekingu) [inline docs/admin/support.rst:84]:
```suggestion
1. Register yourself at https://weblate.org/user/
```

File: docs/admin/support.rst [hunk 7]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
 
    This feature is currently in early beta.
 
-Discover Weblate is an opt-in service that makes it easier for users to find
-Weblate servers and communities. Users can browse registered services on
-<https://weblate.org/discover/>, and find there projects to contribute.
+Discover Weblate is an opt-in service making it easier for users to find
+other Weblate sites and communities. Users can browse registered sites on
+`https://weblate.org/discover <https://weblate.org/discover/>`_, and find
```
Comment (comradekingu) [inline docs/admin/support.rst:65]:
```suggestion
https://weblate.org/discover/ and find
```

File: docs/admin/support.rst (hunk 3 above)
Comment (comradekingu) [inline docs/admin/support.rst:8]:
```suggestion
offerings at https://weblate.org/support.
```

**Comment by comradekingu** [comment]
@nijel OK, thanks.
I thought about whether it should be added here as a requirement if running the latest version
actually was a requirement. From https://weblate.org/nb/news/archive/discover-your-favorite-project/

>The server has to run 4.5.2 (or later once released) to enable the listing.

Wasn't 100% on "released" possibly meaning out of beta.
Just to be clear my understanding is that listed entries aren't removed just because the version is not current.
Would maybe be something to do sorting by. A little incentive to keep the versioning recent, to be listed earlier.

**Comment by comradekingu** [comment]
Neverstale

**Comment by comradekingu** [comment]
Neverstale

**Comment by comradekingu** [comment]
Neverstale

File: docs/admin/support.rst [hunk 8]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
-To list your server with an active support subscription (see
-:ref:`activate-support`) in Discover Weblate all you need to do is turn this on
-in the management panel:
+List your server directly from the management panel if you have an active
+support subscription (:ref:`activate-support`):
 
 .. image:: /images/support-discovery.png
 
-Listing your server without a support subsription in Discover Weblate:
+Listing your server manually:
```
Comment (nijel) [inline docs/admin/support.rst:82]: This is not a more manually that with a subscription, it just needs additional steps.

File: docs/admin/support.rst [hunk 9]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 Getting listed
 ++++++++++++++
 
-.. hint::
+.. note::
 
-   Participating in Discover Weblate makes Weblate submit some information
-   about your server, please see :ref:`support-data`.
+   Make sure you are OK with publishing the info about your Weblate
+   described in :ref:`support-data` for use on Discover Weblate.
```
Comment (nijel) [inline docs/admin/support.rst:74]: Not all submitted data is published.

**Comment by comradekingu** [comment]
Nevestale

**Comment by orangesunny** [comment]
@comradekingu, would you please let me edit the PR, so I can polish it and merge it?

File: docs/admin/support.rst [hunk 10]
```diff
@@ -23,16 +23,16 @@ Basic instance details about your installation are also reported back to Weblate
 
 .. _support-data:
 
-Data submitted to the Weblate
------------------------------
+Info sent to and shown on Weblate.org
+-------------------------------------
```
Comment (nijel) [inline docs/admin/support.rst:27]: Not all data is shown on weblate.org.

File: docs/admin/support.rst [hunk 11]
```diff
@@ -51,46 +51,46 @@ Integration services
...
 
-Listing your server without a support subscription in Discover Weblate:
+All steps for listing a server without a support contract:
 
-1. Register yourself at <https://weblate.org/user/>
-2. Register your Weblate server in the discovery database at <https://weblate.org/subscription/discovery/>
-3. Confirm the service activation in your Weblate and turn on the discovery listing in your Weblate management page using :guilabel:`Enable discovery` button:
+1. Register yourself at https://weblate.org/user/
+2. Register your Weblate site at https://weblate.org/subscription/discovery
+3. Confirm you you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
```
Comment (nijel) [inline docs/admin/support.rst:86]:
```suggestion
3. Confirm you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
```

File: docs/admin/support.rst [hunk 12]
```diff
@@ -51,46 +51,46 @@ Integration services
...
 .. note::
 
    This feature is currently in early beta.
 
-Discover Weblate is an opt-in service that makes it easier for users to find
-Weblate servers and communities. Users can browse registered services on
-<https://weblate.org/discover/>, and find there projects to contribute.
+*Discover* is an opt-in service making it easier for users to find
+other Weblate sites and communities. Users can browse registered sites on
+https://weblate.org/discover and find
```
Comment (nijel) [inline docs/admin/support.rst:65]:
```suggestion
https://weblate.org/discover/ and find
```

File: docs/admin/support.rst [hunk 13]
```diff
@@ -51,46 +51,46 @@ Integration services
 
 .. _discover-weblate:
 
-Discover Weblate
-----------------
+Discover
+--------
```
Comment (nijel) [inline docs/admin/support.rst:55]:
```suggestion
Discover Weblate
----------------
```

File: docs/admin/support.rst [hunk 14]
```diff
@@ -51,46 +51,46 @@ Integration services
...
 .. _customize-discover:
 
-Customizing listing
+Listing description
 +++++++++++++++++++
 
-You can customize the listing by providing a text and image (570 x 260 pixels)
-at <https://weblate.org/user/>.
+Add a text and an image (570 x 260 pixels) about your Weblate site
+from https://weblate.org/user.
```
Comment (comradekingu) [inline docs/admin/support.rst:96]:
```suggestion
from <https://weblate.org/user/>.
```

File: docs/admin/support.rst [hunk 15]
```diff
@@ -23,16 +23,16 @@ Basic instance details about your installation are also reported back to Weblate
 
 .. _support-data:
 
-Data submitted to the Weblate
------------------------------
+Info sent to and shown on Weblate.org
```
Comment (comradekingu) [inline docs/admin/support.rst:26]:
```suggestion
Info sent to and (partially) shown on Weblate.org
```

File: docs/admin/support.rst (hunk 10 above)
Comment (comradekingu) [inline docs/admin/support.rst:27]:
```suggestion
-----------------------------------------------
```

File: docs/admin/support.rst [hunk 16]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 .. _customize-discover:
 
-Customizing listing
+Listing description
 +++++++++++++++++++
 
-You can customize the listing by providing a text and image (570 x 260 pixels)
-at <https://weblate.org/user/>.
+Add a text and an image (570 x 260 pixels) about your Weblate site
+from <https://weblate.org/user/>.
```
Comment (comradekingu) [inline docs/admin/support.rst:96]: This allows the ending dot?

File: docs/admin/support.rst [hunk 17]
```diff
@@ -23,16 +23,16 @@ Basic instance details about your installation are also reported back to Weblate
 
 .. _support-data:
 
-Data submitted to the Weblate
------------------------------
+Info sent to and (partially) shown on Weblate.org
+-----------------------------------------------
```
Comment (comradekingu) [inline docs/admin/support.rst:27]:
```suggestion
-------------------------------------------------
```

File: docs/admin/support.rst [hunk 18]
```diff
@@ -60,37 +60,37 @@ Discover Weblate
...
 .. image:: /screenshots/support-discovery.png
 
-Listing your server without a support subscription in Discover Weblate:
+All steps for listing a server without a support contract:
 
-1. Register yourself at <https://weblate.org/user/>
-2. Register your Weblate server in the discovery database at <https://weblate.org/subscription/discovery/>
-3. Confirm the service activation in your Weblate and turn on the discovery listing in your Weblate management page using :guilabel:`Enable discovery` button:
+1. Register yourself at https://weblate.org/user/
+2. Register your Weblate site at https://weblate.org/subscription/discovery
```
Comment (nijel) [inline docs/admin/support.rst:85]:
```suggestion
2. Register your Weblate site at https://weblate.org/subscription/discovery/
```

File: docs/admin/support.rst (hunk 16 above)
Comment (nijel) [reply docs/admin/support.rst:96]: Yes.

**Comment by nijel** [comment]
@comradekingu Can you please allow editing from maintainers here so that we can address merge conflicts?

File: docs/admin/support.rst [hunk 19]
```diff
@@ -2,10 +2,10 @@ Getting support for Weblate
...
 
 
-Weblate is copylefted libre software with community support.
-Subscribers receive priority support at no extra charge. Prepaid help packages are
-available for everyone. You can find more info about current support
-offerings at <https://weblate.org/support/>.
+*Weblate* is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge.
+Prepaid help packages are available for everyone.
+Find info about current support offerings at https://weblate.org/support.
```
Comment (comradekingu) [inline docs/admin/support.rst:8]: There was some escaping for URLs ending in dots in another PR. Maybe needed here?

File: docs/admin/support.rst [hunk 20]
```diff
@@ -53,37 +53,37 @@ Discover Weblate
 
 .. versionadded:: 4.5.2
 
-Discover Weblate is an opt-in service that makes it easier for users to find
-Weblate servers and communities. Users can browse registered services on
-<https://weblate.org/discover/>, and find there projects to contribute.
+*Discover* is an opt-in service making it easier for users to find
+other Weblate sites and communities.
+Users can browse registered sites and find projects to contribute to on
+https://weblate.org/discover/.
```
Comment (comradekingu) [inline docs/admin/support.rst:59]: There was some escaping for URLs ending in dots in another PR. Maybe needed here?

Comment (nijel) [reply docs/admin/support.rst:59]: https://weblate--5761.org.readthedocs.build/en/5761/ shows how this gets rendered. This is available for all pull request, look for Read the docs in checks

File: docs/admin/support.rst (hunk 10 above)
Comment (orangesunny) [reply docs/admin/support.rst]: solved by rewrite.

File: docs/admin/support.rst (hunk 3 above)
Comment (orangesunny) [reply docs/admin/support.rst]: corrected, changed to URLs only

**Review by orangesunny — APPROVED** [review]
good to go

**Pull request merged**